libs/base/Data/Vect.idr

```diff
@@ -283,7 +286,7 @@ implementation Functor (Vect n) where
 |||
 ||| @ f the partial function (expressed by returning `Maybe`)
 ||| @ xs the vector to check for results
-mapMaybe : (f : a -> Maybe b) -> (xs : Vect n a) -> (m : Nat ** Vect m b)
+mapMaybe : (f : elem -> Maybe b) -> (xs : Vect len elem) -> (m : Nat ** Vect m b)
```

Comment:
The use of `elem` and `b` as opposed to `a` and `b` seems odd to me, given they’re both the element type of different `Vect`s.